utils_nlp/models/transformers/datasets.py

```diff
@@ -391,7 +394,19 @@ def __init__(
                 num_pool=n_processes,
```

Comment:
did you mean CPU in docstring in line 438?
Also, doesn't the number of workers in the data loader take care of parallelization anyway?

Reply:
Hong is the original author of this piece of code. This is the preprocessing part that happens outside the data loader, so I don't think the data loader can take care of this part of parallelization.

Reply:
@saidbleik  you are right. It should be multiple CPUs. @daden-ms can you help fixing this? Thanks!